foreshadow/steps/cleaner.py

```diff
@@ -9,6 +9,50 @@
 from .preparerstep import PreparerStep
 
 
+def _check_empty_columns(X) -> NoReturn:
+    """Check if all columns are empty in the dataframe.
+
+    Args:
+        X: the dataframe
+
+    Returns:
+        the empty columns.
+
+    Raises:
+        ValueError: all columns are dropped.
+
+    """
+    columns = pd.Series(X.columns)
+    empty_columns = columns[X.isnull().all(axis=0).values]
+
+    if len(empty_columns) == len(columns):
+        error_message = (
+            "All columns are dropped since they all have "
+            "over 90% of missing values. Aborting foreshadow."
+        )
+        logging.error(error_message)
+        raise ValueError(error_message)
+    elif len(empty_columns) > 0:
+        logging.info(
+            "Dropping columns due to missing values over 90%: {}"
+            "".format(",".join(empty_columns.tolist()))
+        )
+
+    return empty_columns.tolist()
+
+
+def _abort_if_has_new_empty_columns(current_columns, empty_columns):
```

Comment:
Please remove this step becuase, as discussed, for the columns that happen to be empty in prediction, the chance is that there are null values in this column during training process as well and the identified pipeline should be able to handle it. Think about numerical columns with null values for which pipeline will impute with a mean, for example.

Reply:
Done.